src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -381,7 +381,7 @@ interface SelectedInstitutionFacetButtonProps {
   institutionIdentifier: string;
 }
 
-const SelectedInstitutionFacetButton = ({ institutionIdentifier }: SelectedInstitutionFacetButtonProps) => {
+export const SelectedInstitutionFacetButton = ({ institutionIdentifier }: SelectedInstitutionFacetButtonProps) => {
```

Comment:
Kunne/skulle vært flytta til den nye mappa med `selected_facets`, men tenker det heller kan vente til vi endrer resultatsøk til å bruke det nye oppsettet for fasettknapper